### PR TITLE
blogs page

### LINKED ISSUE #10: Show Blogs page
For url [http://localhost:3000/blog](http://localhost:3000/blog)

### PR DESCRIPTION
**Pull request by promisemukundu**

### CODE REVIEW

File: src/components/Home/LatestBlog.jsx [hunk 1]
```diff
@@ -15,8 +15,8 @@ const LatestBlog = () => {
           q: 'fashion',
           searchIn: 'title',
           sortBy: 'relevancy',
-          pageSize: 3
-        }
+          pageSize: props.pageSize
+       }
       })
       .then((response) => {
         console.log(response);
```
Comment (innoflash) [inline src/components/Home/LatestBlog.jsx:22]: Reduntant log, needs to be removed

File: src/components/Home/LatestBlog.jsx [hunk 2]
```diff
@@ -25,14 +25,15 @@ const LatestBlog = () => {
...
   return (
     <>
       <section className="latest spad">
         <div className="container">
           <div className="row">
             <div className="col-lg-12">
-              <div clasName="section-title">
-                <span>Latest News</span>
+              <div className="section-title">
+                <span>Latest News {}</span>
```
Comment (innoflash) [inline src/components/Home/LatestBlog.jsx:36]: Reduntant curly

**Review by innoflash — COMMENTED** [review]
Work on these

File: src/components/Home/LatestBlog.jsx (hunk 2 above)
Comment (promisemukundu) [reply src/components/Home/LatestBlog.jsx:36]: corrected it Sir

File: src/components/Home/LatestBlog.jsx (hunk 1 above)
Comment (promisemukundu) [reply src/components/Home/LatestBlog.jsx:22]: okay Sir

**Pull request merged**